x-pack/test/cases_api_integration/security_and_spaces/tests/common/internal/bulk_create_attachments.ts

```diff
@@ -376,6 +365,23 @@ export default ({ getService }: FtrProviderContext): void => {
         });
       });
 
+      it('400s when attempting to add more than 100 attachments', async () => {
+        const comment = {
+          type: CommentType.user,
+          comment: 'test',
+          owner: 'securitySolutionFixture',
+        };
+
+        const attachments = Array(101).fill(comment);
```

Comment:
In these integration tests I never know if I wanna just put the value or import the constant(`MAX_BULK_CREATE_ATTACHMENTS`)

Reply:
yeah, but I thought it's good that integration tests fail when constant is changed, reminds us what all places we have put this check 😄

Reply:
Keeps us on edge 😆

Reply:
Genau 😆

Reply:
😆 I tend to user the value for the reasons Janki mentioned.